### DISCUSSION TITLE
A question about  "pg remapped"

### DISCUSSION THREAD

**Discussion by zhucan** [rook/rook#12378] [Q&A]
1. create objectstore 
2. set pg num to "datat" pool of object
3. set autoscale_mode to off
4. create bucket 
5. write data to bucket 
6. the pg will remapp?

![image](https://github.com/rook/rook/assets/13997135/b5010446-4be7-40ae-95f2-2ba9e0da6723)

I want to know the reason about this.

**Comment by BlaineEXE** [comment]
This is a question better asked to the Ceph community since it involves Ceph's pg handling: https://ceph.io/en/community/connect/

**Reply by zhucan** [reply]
I know, but the responses from the ceph community is slowly.
